src/core/Form/Select/BaseSelect/SelectItemList/SelectItemList.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import { default as styled } from 'styled-components';
+import classnames from 'classnames';
+import { SuomifiThemeProp, SuomifiThemeConsumer } from '../../../../theme';
+import { HtmlDivWithRef, HtmlUlWithRef } from '../../../../../reset';
+import { SelectItemProps } from '../SelectItem/SelectItem';
+import { baseStyles } from './SelectItemList.baseStyles';
+
+const baseClassName = 'fi-select-item-list';
+
+const selectItemListClassNames = {
+  content_wrapper: `${baseClassName}_content_wrapper`,
+};
+
+export interface SelectItemListProps {
+  /** SelectItemList container div class name for custom styling. */
+  className?: string;
+  children:
+    | Array<React.ReactElement<SelectItemProps>>
+    | React.ReactElement<SelectItemProps>;
+  forwardRef: React.RefObject<HTMLUListElement>;
+  focusedDescendantId: { id: string };
```

Comment:
Could `focusedDescendantId` be renamed as `focusedDescendant` as it seems little confusing that getting the id from id. This change might make it little more readable.
```tsx
this.props.focusedDescendantId.id
```

Reply:
Good idea. I think this was left here for other reasons e.g. to catch the change in situations where id remains the same but re-render is required. Will change to more simple approach.

Reply:
Should be now changed as proposed.